tests/test_point_source.py

```diff
@@ -1,8 +1,11 @@
 from openmc_plasma_source import FusionPointSource
 
 from openmc import Source
-
+import pytest
 
 def test_creation():
     my_source = FusionPointSource()
     assert isinstance(my_source, Source)
+
+with pytest.raises(ValueError):
+    FusionPointSource(fuel='топливо')
```

Comment:
These should be seperate funcitons
```suggestion
def test_wrong_fuel():
    with pytest.raises(ValueError):
        FusionPointSource(fuel='топливо')
```